<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArduinoShow</class>
 <widget class="QMainWindow" name="ArduinoShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ArduinoShow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="outputPunkte">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="countdown">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>331</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="listButton">
       <property name="text">
        <string>Anschluss</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="connectButton">
       <property name="text">
        <string>Verbinden</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="disconnectButton">
       <property name="text">
        <string>Trennen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="initButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="punktzahl">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>101</width>
      <height>150</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextEdit" name="portNameEdit"/>
     </item>
     <item>
      <widget class="QTextBrowser" name="outputBrowser"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>98</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="erreichte_punktzahl">
       <property name="text">
        <string>Erreichte Punktzahl:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveScore">
       <property name="text">
        <string>Highscore</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="platz_eins">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="platz_zwei">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="platz_drei">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="platz_vier">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="platz_funf">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>34</y>
      <width>81</width>
      <height>213</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="historie">
       <property name="text">
        <string>Historie</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="outputHistorie"/>
     </item>
    </layout>
   </widget>
   <zorder>listButton</zorder>
   <zorder>connectButton</zorder>
   <zorder>initButton</zorder>
   <zorder>disconnectButton</zorder>
   <zorder>portNameEdit</zorder>
   <zorder>outputBrowser</zorder>
   <zorder>erreichte_punktzahl</zorder>
   <zorder>punktzahl</zorder>
   <zorder>saveScore</zorder>
   <zorder>platz_eins</zorder>
   <zorder>platz_zwei</zorder>
   <zorder>platz_drei</zorder>
   <zorder>platz_vier</zorder>
   <zorder>platz_funf</zorder>
   <zorder>outputPunkte</zorder>
   <zorder>countdown</zorder>
   <zorder>outputHistorie</zorder>
   <zorder>historie</zorder>
   <zorder>initButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>513</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
